utils/utils.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 var ADMINS = map[string]bool{
-	"stride1k8c2m5cn322akk5wy8lpt87dd2f4yh9azg7jlh": true, // F5
+	"stride1u20df3trc2c2zdhm8qvh2hdjx9ewh00sv6eyy8": true, // F5
```

Comment:
I think the sed command might be doing this? I haven't changed this manually

Reply:
@sampocs have you seen this behavior using dockernet before? My workflow is
```
make start-docker && make test-integration-docker
make stop-docker
```

Reply:
Now that I think about it - I think it's possible for the new address to persist if one of the build fails (because it exits before it can revert)

Reply:
Will send you the fix offline